cmd/kubectltranslate/kubectltranslate.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	cmdcommon "github.com/konveyor/move2kube/cmd/common"
+	"github.com/konveyor/move2kube/internal/common"
+	"github.com/konveyor/move2kube/internal/move2kube"
+	"github.com/konveyor/move2kube/internal/qaengine"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type translateFlags struct {
+	outpath  string
+	srcpath  string
+	name     string
+	qacaches []string
+}
+
+var verbose bool
+
+func translateHandler(cmd *cobra.Command, flags translateFlags) {
+	// Setup
+	var err error
+	ignoreEnv := false
+	srcpath := flags.srcpath
+	outpath := flags.outpath
+	name := flags.name
+
+	qadisablecli := false
+	qaskip := false
+	qaport := 0
+	qacaches := flags.qacaches
+
+	if srcpath, err = filepath.Abs(srcpath); err != nil {
+		log.Fatalf("Failed to make the source directory path %q absolute. Error: %q", srcpath, err)
+	}
+	if outpath, err = filepath.Abs(outpath); err != nil {
+		log.Fatalf("Failed to make the output directory path %q absolute. Error: %q", outpath, err)
+	}
+
+	// Global settings
+	common.IgnoreEnvironment = ignoreEnv
+	qaengine.StartEngine(qaskip, qaport, qadisablecli)
+	cachepaths := []string{}
+	for i := len(qacaches) - 1; i >= 0; i-- {
+		cachepaths = append(cachepaths, qacaches[i])
+	}
+	qaengine.AddCaches(cachepaths)
+
+	// Plan
+	cmdcommon.CheckSourcePath(srcpath)
+	plan := move2kube.CreatePlan(srcpath, name)
+	outpath = filepath.Join(outpath, plan.Name)
+	cmdcommon.CheckOutputPath(outpath)
+	cmdcommon.CreateOutputDirectoryAndCacheFile(outpath)
+	plan = move2kube.CuratePlan(plan)
+
+	// Translate
+	move2kube.Translate(plan, outpath, qadisablecli)
+	log.Infof("Translated target artifacts can be found at [%s].", outpath)
+}
+
+func main() {
+	// Setup
+	must := func(err error) {
+		if err != nil {
+			panic(err)
+		}
+	}
+	viper.AutomaticEnv()
+
+	flags := translateFlags{}
+	translateCmd := &cobra.Command{
+		Use:   "translate [ -o <output directory> ] [ -n <project name> ] [ -q <list of qacache files> ] -s <source directory>",
```

Comment:
The command should update based on usage. https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/

Reply:
what?